product_docs/docs/pgd/5/reference/testingandtuning.mdx

```diff
@@ -4,117 +4,116 @@ navTitle: Testing and tuning
-
-#### Options to select what to run:
--   `-b, --builtin=NAME[@W]` &mdash; add builtin script NAME weighted at W (default: 1). Use `-b list` to list available scripts.
--   `-f, --file=FILENAME[@W]` &mdash; add script `FILENAME` weighted at W (default: 1)
--   `-N, --skip-some-updates` &mdash;  updates of pgbench_tellers and pgbench_branches. Same as `-b simple-update`
--   `-S, --select-only` &mdash; perform SELECT-only transactions. Same as `-b select-only`
-
-#### Benchmarking options:
--   `-c, --client=NUM` &mdash; number of concurrent database clients (default: 1)
--   `-C, --connect` &mdash; establish new connection for each transaction
--   `-D, --define=VARNAME=VALUE` &mdash;  define variable for use by custom script
--   `-j, --jobs=NUM` &mdash; number of threads (default: 1)
--   `-l, --log` &mdash; write transaction times to log file
--   `-L, --latency-limit=NUM` &mdash; count transactions lasting more than NUM ms as late
--   `-m, --mode=regular|camo|failover` &mdash; mode in which pgbench should run (default: `regular`)
--   `-M, --protocol=simple|extended|prepared` &mdash; protocol for submitting queries (default: `simple`)
--   `-n, --no-vacuum` &mdash; do not run `VACUUM` before tests
--   `-o, --set-option=NAME=VALUE` &mdash; specify runtime SET option
--   `-P, --progress=NUM`  &mdash; show thread progress report every NUM seconds
--   `-r, --report-per-command`  &mdash;  latencies, failures and retries per command
--   `-R, --rate=NUM`  &mdash; target rate in transactions per second
--   `-s, --scale=NUM`  &mdash; report this scale factor in output
--   `-t, --transactions=NUM`  &mdash; number of transactions each client runs (default: 10)
--   `-T, --time=NUM`  &mdash; duration of benchmark test in seconds
--   `-v, --vacuum-all`  &mdash; vacuum all four standard tables before tests
--   `--aggregate-interval=NUM`  &mdash;  data over NUM seconds
--   `--failures-detailed`  &mdash; report the failures grouped by basic types
--   `--log-prefix=PREFIX`  &mdash; prefix for transaction time log file (default: `pgbench_log`)
--   `--max-tries=NUM`  &mdash; max number of tries to run transaction (default: 1)
--   `--progress-timestamp`  &mdash; use Unix epoch timestamps for progress
--   `--random-seed=SEED`  &mdash; set random seed ("time", "rand", integer)
--   `--retry`  &mdash; retry transactions on failover, used with "-m"
--   `--sampling-rate=NUM`  &mdash; fraction of transactions to log (e.g., 0.01 for 1%)
--   `--show-script=NAME`  &mdash; show builtin script code, then exit
--   `--verbose-errors`  &mdash; print messages of all errors
+This option is followed by `NAME=VALUE` entries, which are applied using the 
+PostgreSQL [`SET`](https://www.postgresql.org/docs/current/sql-set.html) command on each server that pgd_bench connects to, and only those servers. 
+
+The other options are identical to the PostgreSQL pgd_bench command. For
```

Comment:
I think this is supposed to say pg_bench - I think the point is the the pgd_bench commands are the same as the pg_bench commands

Reply:
Yes it is, fixed above.